pkg/mons/kubectl.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright © 2023 kubectl-rook-ceph
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package mons
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var script = `
+kubectl -n %s get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data' | tr -d '"' | tr -d '=' | sed 's/[A-Za-z]*//g'
```

Comment:
I wonder if we could covert commands like this to use apis instead of kubectl, instead of delaying the conversion to later. It should be pretty simple in go to read the configmap then parse the data.

Reply:
yeah, we can do, I'll make the changes

Reply:
still need to do this change

Reply:
done